proxy/http/HttpSM.cc

```diff
@@ -464,6 +464,7 @@ HttpSM::attach_client_session(ProxyClientTransaction *client_vc, IOBufferReader
       _client_connection_id = p->connection_id();
     }
   }
+  is_internal = ua_txn->get_netvc()->get_is_internal_request();
```

Comment:
How about:
```
public:
  // Save is is_internal for later logging when UA transaction object no longer exists.
  void save_is_internal() { _is_internal = ua_txn->get_netvc()->get_is_internal_request(); }

  bool is_internal() const { return _is_internal; }

private:
  bool _is_internal;
```

Reply:
At this point I'd think it better to have a "history" subclass that is responsible for gathering this historical information for later use.